### PR TITLE
add nmap rosdep key

### PR DESCRIPTION
**Pull request by Vuwij**
https://apps.fedoraproject.org/packages/nmap
https://packages.debian.org/jessie/nmap
https://packages.ubuntu.com/trusty/nmap
https://packages.gentoo.org/packages/net-analyzer/nmap

### CODE REVIEW

**Review by tfoote — APPROVED** [review]
Oddly packages.ubuntu.com isn't showing any package by that name except cosmic. But that must be a website issue as it's there for all the other versions back to trusty: https://packages.ubuntu.com/cosmic/nmap

```
tfoote@snowman:/tmp/prerelease_job Last: [0] (0s Seconds)
$ rmadison nmap
 nmap | 5.21-1.1ubuntu1 | precise | source, amd64, armel, armhf, i386, powerpc
 nmap | 6.40-0.2ubuntu1 | trusty  | source, amd64, arm64, armhf, i386, powerpc, ppc64el
 nmap | 7.01-2ubuntu2   | xenial  | source, amd64, arm64, armhf, i386, powerpc, ppc64el, s390x
 nmap | 7.60-1ubuntu5   | bionic  | source, amd64, arm64, armhf, i386, ppc64el, s390x
 nmap | 7.60-1ubuntu5   | cosmic  | source, amd64, arm64, armhf, i386, ppc64el, s390x
 nmap | 7.60-1ubuntu5   | disco   | source, amd64, arm64, armhf, i386, ppc64el, s390x
```

File: rosdep/base.yaml [hunk 1]
```diff
@@ -4026,6 +4026,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nmap:
+  debian: [nmap]
```
Comment (nuclearsandwich) [inline rosdep/base.yaml]:
```suggestion
  archlinux: [nmap]
  debian: [nmap]
```
Suggesting the archlinux add while I'm here: https://www.archlinux.org/packages/extra/x86_64/nmap/

**Review by nuclearsandwich — APPROVED** [review]
I've suggested adding the Archlinux value as well. My approval stands for the PR as is _or_ with the incorporation of this suggestion.

File: rosdep/base.yaml (hunk 1 above)
Comment (tfoote) [reply rosdep/base.yaml]: @Vuwij could you accept this change suggestion then we can merge this?

Comment (nuclearsandwich) [reply rosdep/base.yaml]: > @Vuwij could you accept this change suggestion then we can merge this?

In order to merge the change I've committed the addition of the Arch key directly to the PR branch and am taking the above comment as approval of the final state.

**Pull request merged**